composer.json

```diff
@@ -79,5 +79,23 @@
         "mutate": "infection --threads=2 --skip-initial-tests --coverage=build/phpunit",
         "style": "@cs-fix",
         "test": "phpunit"
-    }
+    },
+    "support": {
+        "forum": "https://forum.codeigniter.com",
+        "chat": "https://github.com/codeigniter4/shield/discussions",
+        "slack": "https://codeigniterchat.slack.com",
+        "source": "https://github.com/codeigniter4/shield",
+        "issues": "https://github.com/codeigniter4/shield/issues",
+        "docs": "https://github.com/codeigniter4/shield/blob/develop/docs/index.md"
+    },
+    "funding": [
```

Comment:
While I developed the vast majority of this, I'm not comfortable having my info on there for funding now that it is an official project. Unfortunately we don't currently have any funding options setup for the framework as a whole. Please remove this section.

Reply:
Sorry @lonnieezell , this performance was due to my lack of knowledge.
Done.